rm-datasource/src/main/java/io/seata/rm/datasource/sql/struct/cache/MysqlTableMetaCache.java

```diff
@@ -77,7 +77,7 @@ protected String getCacheKey(Connection connection, String tableName, String res
 
     @Override
     protected TableMeta fetchSchema(Connection connection, String tableName) throws SQLException {
-        String sql = "SELECT * FROM " + ColumnUtils.addEscape(tableName, JdbcConstants.MYSQL) + " LIMIT 1";
+        String sql = "SELECT * FROM " + ColumnUtils.addEscape(tableName, this.getClass().getAnnotation(LoadLevel.class).name()) + " LIMIT 1";
```

Comment:
Generally the information on a class is for external judgment rather than internal use.

Reply:
done